Makefile

```diff
@@ -13,4 +13,4 @@ freeze-requirements:  ## Re-freeze requirements.
 
 .PHONY: snap
 snap:  ## Create a Imagecraft snap
-	@snapcraft clean && snapcraft --use-lxd
+	@snapcraft clean --use-lxd && snapcraft --use-lxd
```

Comment:
I didn't even know that the clean state also required passing the provider to use...

Reply:
I didn't either until someone from the craft team asked if I did that. I have to say I did not checked if this is really making a difference but at least we are "sure" this is cleaning what we are using to build.